collections/_article/dev-snapshot-godot-4-5-beta-1.md

```diff
@@ -0,0 +1,497 @@
+
+#### And more
+
+- Add new `StandardMaterial` properties to allow users to control FPS-style objects (hands, weapons, tools close to the camera). ([GH-93142](https://github.com/godotengine/godot/pull/93142))
+- Fragment density map support. ([GH-99551](https://github.com/godotengine/godot/pull/99551))
+- Implement motion vectors in mobile renderer. ([GH-100283](https://github.com/godotengine/godot/pull/100283))
+- Overhaul the cull mask internals for Lights, Decals, and Particle Colliders. ([GH-102399](https://github.com/godotengine/godot/pull/102399))
+- Various performance optimizations. ([GH-103547](https://github.com/godotengine/godot/pull/103547), [GH-103794](https://github.com/godotengine/godot/pull/103794), [GH-103889](https://github.com/godotengine/godot/pull/103889))
+- Optimize Mobile renderer by using FP16 explicitly. ([GH-107119](https://github.com/godotengine/godot/pull/107119))
+- Fix LightmapGI shadow leaks. ([GH-107254](https://github.com/godotengine/godot/pull/107254))
+
+### XR
+
+The XR team has been doing a [lot of groundwork](https://github.com/godotengine/godot/pulls?q=is%3Amerged+is%3Apr+label%3Atopic%3Axr+milestone%3A4.5+label%3Aenhancement) to support more OpenXR extensions. While it's separate from the main Godot release cycle, the Godot OpenXR Vendors had a [4.0.0 release](https://github.com/GodotVR/godot_openxr_vendors/releases/tag/4.0.0-stable) in April which is worth checking out! [Matthieu Bucchianeri](https://github.com/mbucchia) had a massive first contribution merged to implement support for the Direct3D 12 OpenXR backend ([GH-104207](https://github.com/godotengine/godot/pull/104207)).
+
+[Darío](https://github.com/DarioSamo) and [Logan Lang](https://github.com/devloglogan)'s implementations of respectively fragment density maps and motion vectors for the Mobile renderer also pave the way for better rendering and supporting more extensions for XR.
+
+While those are not merged yet, you can keep an eye on [Bastiaan Olij](https://github.com/BastiaanOlij)'s cutting edge implementations for the OpenXR Spatial Entities ([GH-107391](https://github.com/godotengine/godot/pull/107391)) and Render Model ([GH-107388](https://github.com/godotengine/godot/pull/107388)) extensions, building upon the freshly released OpenXR SDK 1.1.49. [Fredia Huya-Kouadio](https://github.com/m4gr3d) is also putting finishing touches on the support for running hybrid apps from the Godot XR editor ([GH-103972](https://github.com/godotengine/godot/pull/103972)).
```

Comment:
"Those" makes it sound like it's talking about the stuff from the previous paragraph, rather than the current paragraph:

```suggestion
While these are not merged yet, you can keep an eye on [Bastiaan Olij](https://github.com/BastiaanOlij)'s cutting edge implementations for the OpenXR Spatial Entities ([GH-107391](https://github.com/godotengine/godot/pull/107391)) and Render Model ([GH-107388](https://github.com/godotengine/godot/pull/107388)) extensions, building upon the freshly released OpenXR SDK 1.1.49. [Fredia Huya-Kouadio](https://github.com/m4gr3d) is also putting finishing touches on the support for running hybrid apps from the Godot XR editor ([GH-103972](https://github.com/godotengine/godot/pull/103972)).
```